scripts/hash-exec-copy.py

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env python3
+
+import argparse
+from datetime import datetime
+import requests
+import subprocess
```

Comment:
Not that well versed in Python, but I know `datetime` and `subprocess` are included in most Python distributions, however I'm not so sure about `argparse` and `requests`.
Should we have a `requirements.txt` in this directory?

Reply:
Yes! Good point, `requests` isn't python standard library so I can add it to `requirements.txt` :)